paymentsheet/src/main/java/com/stripe/android/link/ui/updatecard/UpdateCardScreenViewModel.kt

```diff
@@ -117,6 +117,7 @@ internal class UpdateCardScreenViewModel @Inject constructor(
                             selectedPaymentDetails = updatedPaymentDetails,
                             linkAccount = account,
                             cvc = null, // CVC is already included in the updated payment details
+                            billingPhoneNumber = cardParams.billingDetails?.phone,
```

Comment:
Add new card: we extract the billing phone from the update params (that's how the UI component bundles it) and pass it to the confirmation flow so that it's passed down in the share endpoint.